clocking/api/view.py

```diff
@@ -82,7 +38,6 @@ def post(self):
                     mac = self.convert_mac_format(mac)
```

Comment:
Use forms for validating and object creation - separate logic between views and forms.

Reply:
Don't take data from request.form. You should take all data from form.data ( as this data is validated and cleaned) .

Reply:
Done.